pkg/features/kube_features.go

```diff
@@ -708,6 +708,11 @@ const (
 	// alpha: v1.20
 	// Adds support for kubelet to detect node shutdown and gracefully terminate pods prior to the node being shutdown.
 	GracefulNodeShutdown featuregate.Feature = "GracefulNodeShutdown"
+	// owner: @janosi
```

Comment:
non-blocking nit: there should be a newline above this line

Reply:
Fixed